--- atuin-18.21.0/Cargo.toml	2006-07-24T01:21:28+00:00
+++ atuin-18.21.0/Cargo.toml	2026-09-03T14:11:46.839499+00:00
@@ -328,11 +328,6 @@
 [target.'cfg(any(target_os = "linux", target_os = "macos", target_os = "windows", target_os = "illumos"))'.dependencies.tree-sitter-powershell]
 version = "0.26.4"
 
-[target.'cfg(any(target_os = "windows", target_os = "macos"))'.dependencies.arboard]
-version = "3.4"
-optional = true
-default-features = false
-
 [target.'cfg(target_os = "linux")'.dependencies.arboard]
 version = "3.4"
 features = ["wayland-data-control"]
@@ -341,10 +336,6 @@
 
 [target."cfg(unix)".dependencies.daemonize]
 version = "0.5.0"
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.61.2"
-features = ["Win32_System_Console"]
 
 [lints.clippy]
 as_ptr_cast_mut = "warn"
@@ -455,3 +446,4 @@
 verbose_bit_mask = "warn"
 while_float = "warn"
 wildcard_imports = "warn"
+
